draft-ietf-netconf-crypto-types.xml

```diff
@@ -255,7 +258,7 @@ Typedefs:
             </ul>
             <t>Each of these groupings are presented in the following subsections.</t>
 
-            <section title='The "encrypted-value-grouping" Grouping' anchor="encrypted-value-grouping">
+            <section title="The &quot;encrypted-value-grouping&quot; Grouping" anchor="encrypted-value-grouping">
```

Comment:
Why is this change this change desired?  

FWIW, I used to use to have it exactly as in your PR, but moved to this approach since 1) it renders the same and 2) looks cleaner in my text-editor.

If not needed, can you please revert (and the other two like it) as it convention is used throughout the suite of client-server drafts.

Reply:
Yes, this is not strictly a necessary change. I only did it for the sake of consistency, since all other such occurrences in the draft use the "&quot;" entity. I can remove it.